### ISSUE TITLE
plan to replace golang_org package

### ISSUE THREAD

**Issue by kamilsk** [Lookyan/netramesh#48]
I suggest to use forks of

- https://github.com/golang/net
- https://github.com/golang/crypto
- https://github.com/golang/text

with patches from `vendor/golang_org` and the possibility of go module to replace dependencies as here

https://github.com/kamilsk/lift/blob/3d70617c288acc83b6bf5e330dd3a3fcb25a2170/go.mod#L10

The final go.mod may looks like

```
require (
	golang.org/x/crypto ...
	golang.org/x/net ...
	golang.org/x/text ...
)

replace golang.org/x/crypto => github.com/Lookyan/crypto v0.0.1-patch
replace golang.org/x/net => github.com/Lookyan/net v0.0.1-patch
replace golang.org/x/text => github.com/Lookyan/text v0.0.1-patch
```

**Comment by kamilsk** [comment]
It will make possible to close #35, show difference between patched versions and upstream, fix `go get -u` to update deps (when vendor/golang_org disappears)

**Comment by kamilsk** [comment]
I can take care of routine: create forks and branches with patches, etc

**Comment by Lookyan** [comment]
I hope fasthttp will eventually replace the current fork of stdlib. But yes, for now, we can handle it in a better way. 
Not sure we need to do it right now, but if you think it is handy you can do it.

**Comment by kamilsk** [comment]
fixed by bc4947a
